R/estimate_secondary.R

```diff
@@ -133,7 +135,8 @@
 #'
 #' options(old_opts)
 #' }
-estimate_secondary <- function(reports,
+estimate_secondary <- function(data,
+                               reports,
```

Comment:
same as above re positioning

Reply:
Fixed in a3b541678124adc78310aa8ae14851611aead1a6.